guide/src/advanced/troubleshooting/help-command.md

```diff
@@ -14,17 +14,7 @@ hermes help create
 will provide details about all the valid invocations of the `create` CLI command.
 
 ```
-USAGE:
```

Comment:
I'm running `mdbook serve` locally from this branch, and the output seems incomplete. Any ideas?

> http://localhost:3000/advanced/troubleshooting/help-command.html

![Screenshot 2022-09-14 at 15 21 27](https://user-images.githubusercontent.com/1757002/190165446-71eec153-0f52-4754-bbf7-5d3baf758775.png)

Reply:
Hey, I think you need to install `mdbook-template` with `cargo install mdbook-template`.